openg2p_disbursement/models/openg2p_disbursement_batch_transaction.py

```diff
@@ -217,78 +219,80 @@ def create_bulk_transfer(self):
                 [("batch_id", "=", self.id)], limit=limit, offset=offset
             )
 
-        url_token = "http://identity.ibank.financial/oauth/token"
-
-        headers_token = {
-            "Platform-TenantId": "ibank-usa",
-            "Authorization": "Basic Y2xpZW50Og==",
-            "Content-Type": "text/plain",
-        }
-        params_token = {
-            "username": os.environ.get("username"),
-            "password": os.environ.get("password"),
-            "grant_type": os.environ.get("grant_type"),
-        }
-
-        try:
-            response_token = requests.request(
-                "POST", url_token, headers=headers_token, params=params_token
-            )
-
-            response_token_data = response_token.json()
-            self.token_response = response_token_data["access_token"]
-
-        except BaseException as e:
-            _logger.exception(e)
-
         # Uploading to AWS bucket
         uploaded = self.upload_to_aws(csvname, "paymenthub-ee-dev")
 
-        headers = {"Platform-TenantId": "ibank-usa"}
+        # bulk transfer
+        url = "http://bulk-connector.sandbox.fynarfin.io/batchtransactions"
```

Comment:
We should not hardcode URLs in the code.

Reply:
Ok will add environment variables for tenant.